packages/components/src/logos/jetpack-logo/index.tsx

```diff
@@ -24,79 +23,109 @@ type JetpackLogoProps = {
 	 * @default false
 	 */
 	monochrome?: boolean;
+	/**
+	 * Theme of the logo.
+	 * - `default`: use the default theme (ie. inherit).
+	 * - `light`: render.
+	 * - `dark`: use the dark theme.
+	 * @default 'default'
+	 */
```

Comment:
I'm not sure I fully understand why we need a `theme` prop instead of just using `currentColor` for the "foreground" parts (with the "cutout" being transparent in the monochrome case). Is that part of what we're figuring out with the brand designers right now?

Reply:
The monochrome version having a cutout vs the brand color version being filled white is part of the logo specs (see TTzDk3dzct551hYWRXPFa3-fi-2395_14811 ):
- branded color, light theme: white triangles
- branded color, dark theme: white triangles
- monochrome, light or dark: cutout

More in general, I added the "theme" prop as a way to generate all possible combinations from the specs. Especially today, we wouldn't have a way to render the logo accordingly on a light/dark background.

When we add a `Theme` component, we may soft deprecate the prop and recommend wrapping the logos in a `Theme` component instead

Reply:
We can't use currentColor text in the branded color version?